queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name endswith 'mon';
+SELECT * from animals WHERE date_of_birth > '2016' AND date_of_birth < '2019';
+SELECT * from animals WHERE escape_attempts < 3 AND neutered = true;
+SELECT * from animals WHERE name = 'Agumon' OR name = 'Pikachu';
```

Comment:
- [ ] in this query you should only return the date of birth not the data from all columns please replace the * with the proper word.